components/fortune.tsx

```diff
@@ -106,7 +115,15 @@ const Fortune: FC<{ showFortune: boolean }> = ({ showFortune }) => {
       <Heading fontSize="24px" mb="10px" color="#4C6711">
         今日の運勢は{fortune.type}なのだ！
       </Heading>
-      <Text>{fortune.message}</Text>
+      <Text mb="20px">{fortune.message}</Text>
```

Comment:
`px`値を指定するより、トークンを指定する方が適切です。

https://yamada-ui.com/ja/styled-system/theming/default-theme#spaces

```suggestion
      <Text mb="5">{fortune.message}</Text>
```